pkgs/applications/networking/cluster/vcluster/default.nix

```
@@ -0,0 +1,46 @@
+{ lib, buildGo117Module, fetchFromGitHub, installShellFiles }:
+
+buildGo117Module rec {
+  pname = "vcluster";
+  version = "0.5.3";
+  tag = "v${version}";
+
+  src = fetchFromGitHub {
+    owner = "loft-sh";
+    repo = pname;
+    rev = tag;
```

```suggestion

  src = fetchFromGitHub {
    owner = "loft-sh";
    repo = pname;
    rev = "v${version}";
```
Tag also isn't used outside the fetcher, and this is rarely done - better to avoid it